src/main/java/com/mercadolibre/grupo1/projetointegrador/dtos/SectionDTO.java

```diff
@@ -0,0 +1,19 @@
+package com.mercadolibre.grupo1.projetointegrador.dtos;
+
+import lombok.Data;
+
+import javax.validation.constraints.NotNull;
+
+/**
+ * @author Ederson Rodrigues Araujo
+ * DTO pegando o id da Section e o id da Warehouse
+ */
+
+@Data
+public class SectionDTO {
+
+    @NotNull
```

Comment:
Interessante a gente já ir botando as mensagens de validacao. Tipo: "Codigo da sessao nao pode ser nulo"

```suggestion
    @NotNull(message = "Código da sessão não pode ser nulo")
```

Reply:
concordo com vc, obrigado pelo feedback